pkg/controller/volume/attachdetach/populator/desired_state_of_world_populator.go

```diff
@@ -113,3 +125,99 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		dswp.desiredStateOfWorld.DeletePod(dswPodUID, dswPodToAdd.VolumeName, dswPodToAdd.NodeName)
 	}
 }
+
+func (dswp *desiredStateOfWorldPopulator) isPodTerminated(pod *v1.Pod) bool {
```

Comment:
Instead of duplicating this logic here and in `pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go#L141`, move the logic into a common "util" package (maybe `pkg/volume/util/util.go` or `pkg/volume/util/volumehelper/volumehelper.go`). That way if someone finds a bug with this logic one fix will fix it in both places.